spec-main/api-menu-spec.ts

```diff
@@ -822,6 +822,31 @@ describe('Menu module', function () {
         menu.closePopup()
       })
     })
+
+    it('prevents menu from getting garbage-collected when popuping', (done) => {
+      const menu = Menu.buildFromTemplate([{ role: 'paste' }])
+      menu.popup({ window: w })
+
+      // Keep a weak reference to the menu.
+      const v8Util = process.electronBinding('v8_util')
+      const map = (v8Util as any).createIDWeakMap() as any
```

Comment:
can you add `createIDWeakMap()` to `V8UtilBinding` in `typings/internal-ambient.d.ts` instead?

Reply:
ref #21171

Reply:
let's merge this one first, I will fix this in the other PR